lib/iris/tests/integration/test_regridding.py

```diff
@@ -59,11 +59,11 @@ def _regrid(self, method):
 
     def test_linear(self):
         res = self._regrid("linear")
-        self.assertArrayShapeStats(res, (73, 96), -16100.351951, 5603.850769)
+        self.assertArrayShapeStats(res, (73, 96), 17799.296120, 11207.701323)
```

Comment:
I'm going to trust Stephen on these changes, I haven't looking into where they are coming from myself.